effect/_intents.py

```diff
@@ -29,12 +29,8 @@ class ParallelEffects(object):
     An effect intent that asks for a number of effects to be run in parallel,
     and for their results to be gathered up into a sequence.
 
-    Performers for this intent could perform the child effects in threads, or
```

Comment:
I think this warning should remain. Twisted performers will still interact badly with a thread-dispatching `perform_parallel`. (Calling the reactor from a thread).